### DISCUSSION TITLE
[Component] [tree-select] TreeSelect 在文档中的分类应当为 Form 表单组件

### DISCUSSION THREAD

**Discussion by AnzhiZhang** [element-plus/element-plus#16996] [Documentation]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.3.4`
- Element Plus Version: `2.3.9`
- Browser / OS: `*`
- Build Tool: `Vite`

### Related Component

- `el-tree-select`

### Reproduction Link

[Link]()

### Steps to reproduce

1

### What is Expected?

.

### What is actually happening?

.

### Additional comments

_(empty)_

**Comment by btea** [comment]
cc @kooriookami

**Reply by kooriookami** [reply]
I agree.

**Reply by btea** [reply]
I created a PR. #17005
